src/main/resources/migration/tenant/4.0-2020-03-18-OC-12403.xml

```diff
@@ -95,13 +95,5 @@
         </sql>
     </changeSet>
 
-    <changeSet author="htaycher " id="2020-03-18-12403-02">
-        <sql splitStatements="false">
-            CREATE TRIGGER study_event_insert
-            AFTER INSERT ON study_event
-            FOR EACH ROW
-            EXECUTE PROCEDURE study_event_trigger_new();
-        </sql>
-    </changeSet>
```

Comment:
Why is this trigger being removed? Since this was already run on the dev servers, are you planning on manually removing the triggers on those servers?

Reply:
pulling out